PyPDF2/_merger.py

```diff
@@ -649,7 +649,7 @@ def add_outline_item(
         if writer is None:
             raise RuntimeError(ERR_CLOSED_WRITER)
         return writer.add_outline_item(
-            title, pagenum, parent, color, bold, italic, fit, *args
+            title, pagenum, parent, None, color, bold, italic, fit, *args
```

Comment:
Why is this necessary?

Reply:
add_outline now proposes Before parameter (to be set to None if you wan to append at the end of the list). Although the parameter is fully optional, I prefer to be clear